### ISSUE TITLE
[$40]{MSFT-101} Challenges page: The title of each challenge under the 'open challenges' and 'ongoing challenges' should be designated with heading level 3 to help in quick navigation for screen reader users.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2805]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure the screen reader is switched on. 3. Pre the letter 'h' to reach the heading 'open challenges' 4. Press the down arrow key to read through the multiple challenges on the page. 5. Similarly press the letter 'h' to move to the heading 'ongoing challenges'. 6. Press the down arrow key to read through the challenges in this section. 7. Notice that there is no mechanism to skip from one challenge to the other.

**Expected Result**
The title of each challenge should be designated with a level 3 heading so as to facilitate quick navigation for screen reader users.

**Actual Result**
Screen readers do not have a mechanism to skip from one challenge to the other. They will have to keep pressing the down arrow to read through the entire list of challenges.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: AA
**WCAG Success Criterion**: 2.4.6 Headings and Labels
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%208

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096059 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096059 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096059 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096059 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097279 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097279 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097279 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097279<br/><br/>```This is an automated message for crazyk via Topcoder X```
